integrations/docker/images/chip-build-mbed-os/Dockerfile

```diff
@@ -15,21 +15,10 @@ RUN set -x \
     && rm -rf /var/lib/apt/lists \
     && : # last line
 
-# ------------------------------------------------------------------------------
-# Install Mbed-OS sources
-RUN set -x \
-    && (mkdir -p /opt/mbed-os \
-    && cd /opt/mbed-os \
-    && wget --progress=dot:giga https://github.com/ARMmbed/mbed-os/archive/mbed-os-6.7.0.tar.gz \
-    && tar --strip-components=1 -xzf mbed-os-6.7.0.tar.gz \
-    && rm mbed-os-6.7.0.tar.gz) \
-    && : # last line
-
 # ------------------------------------------------------------------------------
 # Install Python modules
 RUN set -x \
     && pip3 install --no-cache-dir -U mbed-cli mbed-tools \
-    && pip3 install --no-cache-dir -r /opt/mbed-os/requirements.txt \
```

Comment:
How do we install mbed os requirements in the docker image ?

Reply:
Installing mbed-cli and mbed-tools should be enough for setup Python environment

Reply:
The system does not complain when we compile ? I was thinking mbed cli tool does not automaticaly install `mbed-os/requirements.txt`

Reply:
As far as I remember, there were no problems with that. I think the entire requirements for mbed-tools, which we directly use, are installed properly.